app/models/book_suggestion.rb

```diff
@@ -0,0 +1,7 @@
+class BookSuggestion < ApplicationRecord
+  validates :author, presence: true
+  validates :title, presence: true
+  validates :link, presence: true
+
+  belongs_to :user
```

Comment:
Does the user have book suggestions?

Reply:
Yes, it does! https://trello.com/c/dktKCkPw

`user_id: references`

While the suggestion can be made without an user, it could also be made with one.

If not, then, the user_id field is wrong on the card.